- Use EEPROM_SFF_8436 to access QSFP EEPROM

From: Shuotian Cheng <shuche@microsoft.com>

CONFIG_EEPROM_SFF_8436=m

- Maxim MAX6620 sensor chip driver
CONFIG_SENSORS_MAX6620=m

- Delta DPS460 driver
CONFIG_PMBUS=m
CONFIG_SENSORS_PMBUS=m
CONFIG_SENSORS_DNI_DPS460=m

- Set I2C_MUX_GPIO, GPIO_SYSFS, and GPIO_SCH to enable platform info access
CONFIG_I2C_MUX_GPIO=m
CONFIG_GPIO_SYSFS=y
CONFIG_GPIO_SCH=m

- Unset X86_PAT and STRICT_DEVMEM according to Broadcom's requirement
CONFIG_X86_PAT is not set
CONFIG_STRICT_DEVMEM is not set
---
 debian/build/build_amd64_none_amd64/.config |   13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
index e927c8d..74d6692 100644
--- a/debian/build/build_amd64_none_amd64/.config
+++ b/debian/build/build_amd64_none_amd64/.config
@@ -580,8 +580,7 @@ CONFIG_MTRR=y
 CONFIG_MTRR_SANITIZER=y
 CONFIG_MTRR_SANITIZER_ENABLE_DEFAULT=0
 CONFIG_MTRR_SANITIZER_SPARE_REG_NR_DEFAULT=1
-CONFIG_X86_PAT=y
-CONFIG_ARCH_USES_PG_UNCACHED=y
+# CONFIG_X86_PAT is not set
 CONFIG_ARCH_RANDOM=y
 CONFIG_X86_SMAP=y
 CONFIG_X86_INTEL_MPX=y
@@ -2036,6 +2035,7 @@ CONFIG_EEPROM_LEGACY=m
 CONFIG_EEPROM_MAX6875=m
 CONFIG_EEPROM_93CX6=m
 # CONFIG_EEPROM_93XX46 is not set
+CONFIG_EEPROM_SFF_8436=m
 CONFIG_CB710_CORE=m
 # CONFIG_CB710_DEBUG is not set
 CONFIG_CB710_DEBUG_ASSUMPTIONS=y
@@ -3567,7 +3567,7 @@ CONFIG_I2C_MUX=m
 #
 # Multiplexer I2C Chip support
 #
-# CONFIG_I2C_MUX_GPIO is not set
+CONFIG_I2C_MUX_GPIO=m
 # CONFIG_I2C_MUX_PCA9541 is not set
 # CONFIG_I2C_MUX_PCA954x is not set
 # CONFIG_I2C_MUX_PINCTRL is not set
@@ -3743,7 +3743,7 @@ CONFIG_GPIO_AMDPT=m
 #
 # CONFIG_GPIO_F7188X is not set
 # CONFIG_GPIO_IT87 is not set
-# CONFIG_GPIO_SCH is not set
+CONFIG_GPIO_SCH=m
 # CONFIG_GPIO_SCH311X is not set
 
 #
@@ -3911,6 +3911,7 @@ CONFIG_SENSORS_MAX1668=m
 CONFIG_SENSORS_MAX6639=m
 CONFIG_SENSORS_MAX6642=m
 CONFIG_SENSORS_MAX6650=m
+CONFIG_SENSORS_MAX6620=m
 # CONFIG_SENSORS_MAX6697 is not set
 # CONFIG_SENSORS_MAX31790 is not set
 # CONFIG_SENSORS_MCP3021 is not set
@@ -3949,6 +3950,7 @@ CONFIG_SENSORS_LM25066=m
 # CONFIG_SENSORS_MAX16064 is not set
 # CONFIG_SENSORS_MAX20751 is not set
 # CONFIG_SENSORS_MAX34440 is not set
+CONFIG_SENSORS_DNI_DPS460=m
 # CONFIG_SENSORS_MAX8688 is not set
 # CONFIG_SENSORS_TPS40422 is not set
 CONFIG_SENSORS_TPS53679=m
@@ -7536,8 +7538,7 @@ CONFIG_ARCH_HAS_UBSAN_SANITIZE_ALL=y
 # CONFIG_ARCH_WANTS_UBSAN_NO_NULL is not set
 # CONFIG_UBSAN is not set
 CONFIG_ARCH_HAS_DEVMEM_IS_ALLOWED=y
-CONFIG_STRICT_DEVMEM=y
-CONFIG_IO_STRICT_DEVMEM=y
+# CONFIG_STRICT_DEVMEM is not set
 # CONFIG_X86_VERBOSE_BOOTUP is not set
 CONFIG_EARLY_PRINTK=y
 # CONFIG_EARLY_PRINTK_DBGP is not set
